cmp/grid/GridModel.js

```diff
@@ -191,6 +195,7 @@ export class GridModel {
 
         Object.assign(this, rest);
 
+        this.defaultCol = withDefault(defaultCol, {});
```

Comment:
In this case, could do the defaulting in the constructor for self-documenting....also more in-line with the other two defaults.